scripts/supercloud/run_action_policy_experiments.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
could you add the results of this experiment to this PR? (can be done after merging too.) it helps keep things organized & lets us easily look back to performance of old things that were tried